datahub-web-react/src/app/search/SearchResults.tsx

```diff
@@ -125,24 +162,43 @@ export const SearchResults = ({
     const searchResultUrns = combinedSiblingSearchResults.map((result) => result.entity.urn) || [];
     const selectedEntityUrns = selectedEntities.map((entity) => entity.urn);
 
+    const { isOpen: isSidebarOpen, toggle: toggleSidebar } = useToggle({
+        initialValue: true,
+        onToggle: (isNowOpen: boolean) =>
+            analytics.event({
+                type: EventType.BrowseV2ToggleSidebarEvent,
+                action: isNowOpen ? 'open' : 'close',
+            }),
+    });
+
     return (
         <>
             {loading && <Message type="loading" content="Loading..." style={{ marginTop: '10%' }} />}
-            <div>
+            <SearchResultsWrapper showUpdatedStyles={showSearchFiltersV2}>
                 <SearchBody>
-                    <div id={SEARCH_RESULTS_FILTERS_ID}>
-                        <SearchFiltersSection
-                            filters={filters}
-                            selectedFilters={selectedFilters}
-                            unionType={unionType}
-                            loading={loading}
-                            onChangeFilters={onChangeFilters}
-                            onChangeUnionType={onChangeUnionType}
-                        />
-                    </div>
-                    <ResultContainer>
+                    {!showSearchFiltersV2 && (
+                        <div id={SEARCH_RESULTS_FILTERS_ID} data-testid="search-filters-v1">
+                            <SearchFiltersSection
+                                filters={facets}
+                                selectedFilters={selectedFilters}
+                                unionType={unionType}
+                                loading={loading}
+                                onChangeFilters={onChangeFilters}
+                                onChangeUnionType={onChangeUnionType}
+                            />
+                        </div>
+                    )}
+                    {showBrowseV2 && (
```

Comment:
What happens if browseV2 is enabled, but searchV2 is disabled? Does this still work as expected or do we have a strange state?

Reply:
definitely a strange state. things would "work" but you'd have two sidebars - filters and browse and it would look super wonky. definitely something we need to call out in release that this is a bad state. you can have browse off, both off, or both on, but not browse on and search filters off